app/workers/builds.php

```diff
@@ -39,7 +39,7 @@ public function run(): void
     {
         $type = $this->args['type'] ?? '';
         $projectId = $this->args['projectId'] ?? '';
-        $functionId = $this->args['functionId'] ?? '';
+        $functionId = $this->args['resourceId'] ?? '';
```

Comment:
```suggestion
        $functionId = $this->args['functionId'] ?? '';
```
At the moment, the controller is passing `functionId` as key and not `resourceId` yet. So let's keep it that way .